frontend/talentsearch/src/js/components/search/SearchContainer.test.tsx

```diff
@@ -51,6 +51,7 @@ const poolCandidateResults = [
 const totalCandidateCount = 10;
 
 describe("SearchContainer", () => {
+  jest.setTimeout(10000); // TODO: remove in #4755
```

Comment:
Is this **TODO** related to these test failures?

![Screen Shot 2022-12-20 at 19 19 10](https://user-images.githubusercontent.com/3046459/208791760-2083e644-e8e3-4557-9433-294381869b86.png)

Reply:
Yeah, I have no idea why but this test, along with the experience form, are timing out. I'm not sure why this one specifically started to time out after the change 😢